.github/workflows/docker.yml

```diff
@@ -0,0 +1,52 @@
+name: Docker Image
+
+on:
+  push:
+    branches: [ development ]
+  pull_request:
+    branches: [ development ]
```

Comment:
@pdschubert  
I'm not sure if you really want that. From what I know, this means every PR (before approval and merging) will build a new docker image an push it into the registry. Even for every new commit on a open PR this will build and upload a new image.

Reply:
oh sorry that is a leftover from testing the change to do the multi platform build. I'll remove this in a moment when fixing the whitespaces.
sorry testing CI updates is always painful and I needed a deployed image to verify if it actually works on multiple platforms